package.json

```diff
@@ -36,5 +36,8 @@
     "hooks": {
       "pre-commit": "./scripts/hooks/apply-formatters.sh && ./scripts/hooks/remove-test-prefixes.sh && ./scripts/hooks/generate-docu.sh && ./scripts/hooks/prevent-illegal-characters.sh"
     }
+  },
+  "dependencies": {
+    "i": "^0.3.6"
```

Comment:
Is this package installed intentionally?
```suggestion
```

Reply:
Nope I will remove it... thanks